tests/ui-toml/suppress_lint_in_const/clippy.toml

```diff
@@ -0,0 +1 @@
+suppress-restriction-lint-in-const = false
```

Comment:
This test should test the non default value. Could you set it to true and make sure to include all the tests from `tests/ui/indexing_slicing_index.rs` which were previously suppressed?